src/main/java/io/jenkins/plugins/pipelinegraphview/consoleview/PipelineConsoleViewAction.java

```diff
@@ -55,35 +55,44 @@ public String getIconClassName() {
   public HttpResponse getSteps(StaplerRequest req) throws IOException {
     String nodeId = req.getParameter("nodeId");
     if (nodeId != null) {
-      logger.debug("getSteps was passed nodeId '" + nodeId + "'.");
-      ObjectMapper mapper = new ObjectMapper();
-      PipelineStepList steps = stepApi.getSteps(nodeId);
-      String stepsJson = mapper.writeValueAsString(steps);
-      if (logger.isDebugEnabled()) {
-        logger.debug("Steps: '" + stepsJson + "'.");
-      }
-      return HttpResponses.okJSON(JSONObject.fromObject(stepsJson));
+      return HttpResponses.okJSON(getSteps(nodeId));
     } else {
       return HttpResponses.errorJSON("Error getting console text");
     }
   }
 
+  // Private method for testing.
```

Comment:
the method is protected? am I missing something I don't see it getting called in a test?